### PR TITLE
Update dlp-sensitivity-label-as-condition.md

### PR DESCRIPTION
**Pull request by cpjopp**
The locations mentioned in the article are confusing / incomplete. The article only states Windows 10 devices  for the Endpoint DLP locations. Windows 11 and MacOS should be added - or the locations should be replaced with Endpoint DLP with a link to the supported OS Versions for Endpoint DLP

### CODE REVIEW

File: microsoft-365/compliance/dlp-sensitivity-label-as-condition.md [hunk 1]
```diff
@@ -30,7 +30,9 @@ You can use [sensitivity labels](sensitivity-labels.md) as a condition in DLP po
 - Exchange Online email messages
 - SharePoint Online
 - OneDrive for Business sites
-- Windows 10 devices
+- Windows 10 devices (EndPoint DLP)
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/dlp-sensitivity-label-as-condition.md]:
```suggestion
- Windows 10 devices (Endpoint DLP)
```

File: microsoft-365/compliance/dlp-sensitivity-label-as-condition.md [hunk 2]
```diff
@@ -30,7 +30,9 @@ You can use [sensitivity labels](sensitivity-labels.md) as a condition in DLP po
...
+- Windows 11 devices (EndPoint DLP)
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/dlp-sensitivity-label-as-condition.md]:
```suggestion
- Windows 11 devices (Endpoint DLP)
```

File: microsoft-365/compliance/dlp-sensitivity-label-as-condition.md [hunk 3]
```diff
@@ -30,7 +30,9 @@ You can use [sensitivity labels](sensitivity-labels.md) as a condition in DLP po
...
+- MacOS devices (EndPoint DLP)
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/dlp-sensitivity-label-as-condition.md]:
```suggestion
- MacOS devices (Endpoint DLP)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @cpjopp, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@chrfox Please merge this PR. Thanks

**Comment by chrfox** [comment]
query out to PG

**Pull request merged**